clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerMembershipManagerTest.java

```diff
@@ -1617,10 +1617,25 @@ public void testRevokePartitionsUsesTopicNamesLocalCacheWhenMetadataNotAvailable
     }
 
     @Test
-    public void testOnSubscriptionUpdatedTransitionsToJoiningOnlyIfNotInGroup() {
+    public void testOnSubscriptionUpdatedDoesNotSetShouldTransitionToJoiningIfInGroup() {
```

Comment:
there is no "shouldTransitionToJoining" var/concept anymore, so should we update this ? maybe simply `...DoesNotTransitionToJoiningIfInGroup`

Reply:
Updated the test case name to `testOnSubscriptionUpdatedDoesNotTransitionToJoiningIfInGroup`.